src/js/view/view.js

```diff
@@ -716,8 +716,15 @@ function View(_api, _model) {
             default:
                 if (_captionsRenderer) {
                     _captionsRenderer.show();
-                    if (state === STATE_PAUSED && _controls && !_controls.showing) {
-                        _captionsRenderer.renderCues(true);
+                    if (state === STATE_PLAYING) {
+                        if (_model.get('streamType') === 'LIVE') {
+                            _captionsRenderer.clear();
+                        }
+                    }
+                    if (state === STATE_PAUSED) {
+                        if (_controls && !_controls.showing) {
+                            _captionsRenderer.renderCues(true);
+                        }
```

Comment:
I don't think the issue we're trying to fix is related to the captions renderer updating in relation to the player state. We should be able to address this issue entirely in the provider where we are seeing the issue.